iris_grib/_save_rules.py

```diff
@@ -315,9 +315,9 @@ def latlon_points_irregular(cube, grib):
     # So, this only works at present if the x and y dimensions are **equal**.
     lon_values = x_coord.points / _DEFAULT_DEGREES_UNITS
     lat_values = y_coord.points / _DEFAULT_DEGREES_UNITS
-    eccodes.codes_set_array(grib, 'longitudes',
+    eccodes.codes_set_array(grib, 'longitude',
                             np.array(np.round(lon_values), dtype=np.int64))
-    eccodes.codes_set_array(grib, 'latitudes',
+    eccodes.codes_set_array(grib, 'latitude',
```

Comment:
I'm not able to easily pin-point the [change in eccodes](https://confluence.ecmwf.int/display/ECC/History+of+Changes) that is the source of this difference :disappointed:

Reply:
> "why?"

At a guess, they are trying to increase consistency between different templates.
But clearly, only in an incremental way.  
Which makes rather a mess -- and they seem to not really care about the difference between a behavioural enhancement and a "breaking" change.

I guess they _could_ have supported either key on anything, as your change implements
😪

Reply:
@pp-mo BTW this is the **breaking change** in the `eccodes` API that is forcing us to introduce a minimum pin.

i.e., `longitude`/`latitude` for `codes_set_array` works for `python-eccodes>=1.6.1,eccodes>=2.32.1`, otherwise it must be `longitudes`/`latitudes`.

Reply:
Understood.
We just suck it up -- no real choice !